### PR TITLE
[SDK-593] Add permission and orientation fix to photo capture element

### PR DESCRIPTION
**Pull request by rYuuk**
## Description

- Added fetching of camera permission for ios and android
- Also fixed orientation of captured image

## How to Test

-   Add steps to locally test these changes

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.

[SDK-593]: https://ready-player-me.atlassian.net/browse/SDK-593?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Runtime/AvatarCreator/UI/PhotoCaptureElement.cs [hunk 1]
```diff
@@ -64,6 +85,35 @@ public void TakePhoto()
             onPhotoCaptured?.Invoke(texture);
         }
 
+        private async Task GetPermission()
```
Comment (HarrisonHough) [inline Runtime/AvatarCreator/UI/PhotoCaptureElement.cs:88]: more of a question but if they aren't targeting mobile then this function essentially does nothing? 

Is there some sort of permission API for Standalone/desktop Windows and Mac? 🤔

Comment (rYuuk) [reply Runtime/AvatarCreator/UI/PhotoCaptureElement.cs:88]: I didn't found any such settings in build setting or api for unity

Comment (HarrisonHough) [reply Runtime/AvatarCreator/UI/PhotoCaptureElement.cs:88]: cool just wondering :D

**Review by HarrisonHough — APPROVED** [review]
Code looks good and seems to work, awesome stuff!

**Pull request merged**